### ISSUE TITLE
Do not depend on host to host transfers for Ceph MV action

### ISSUE THREAD

**Issue by OpenNebulaSupport** [OpenNebula/one#5653]
**Description**
Do not depend on host to host transfers for Ceph MV action

**Use case**
In some circumstances, when using [VM HA](https://docs.opennebula.io/6.2/installation_and_configuration/ha/vm_ha.html), VMs cannot be rescheduled because SSH becomes non accessible and you can see an error like this one:

```
ERROR: mv: Command "set -e -o pipefail tar -C /var/lib/one//datastores/<DS_ID> --sparse -cf - 0 | ssh <HOST> 'tar -C /var/lib/one//datastores/<DS_ID> --sparse -xf Error copying disk directory to target hostailed: ssh: connect to host <HOST> port 22: No route to host".
```

**Interface Changes**
Ceph driver

## Progress Status
- [ ] Branch created
- [ ] Code committed to development branch
- [ ] Testing - QA
- [ ] Documentation
- [ ] Release notes - resolved issues, compatibility, known issues
- [ ] Code committed to upstream release/hotfix branches
- [ ] Documentation committed to upstream release/hotfix branches

**Comment by rdziwinski** [comment]
Hello, any information when this fix will be get to community version?

**Comment by rsmontero** [comment]
It is in 6.4 beta and will be released in 6.4 this month

However the patch is straightforward, just need to replace the ceph/mv action. This can be safely applied to any 6.2 installation
